tophat/RichTextView
# fix: custom HTML color

## Description
added `replaceColor` function

## DevQA

### DevQA Prep
- add this to the examples `<span style="color:rgb(33, 102, 216);">“The 9/11 Commission Report, completed in 2004 by a team appointed by President Bush and members of Congress, recommended a series of changes in government policy to respond to the threat of modern terrorism.”&nbsp;</span>`
- Run `pod install`
- Test `RichTextView` to ensure that it passes (unit and UI tests)
- Navigate to the `Example` project in the root `Example` folder and run the app
- Ensure that everything in the `Example` app looks visually correct


### DevQA Steps
run the example project and check if the string is blue. 



### Comments
